IDEAS/Buildings/Components/Window.mo

```diff
@@ -45,6 +53,13 @@ model Window "Multipane window"
         extent={{10,-10},{-10,10}},
         rotation=-90,
         origin={-40,-100})));
+
+  Modelica.Thermal.HeatTransfer.Components.ThermalConductor theBri(final G=briType.G) if
+       briType.present "Themal bridge of the window perimeter"
```

Comment:
Would it not be better to recompute the value of the existing thermal resistor based on the value of `briType.G`? The current implementation is computationally more difficult and adds a graphic element that may better be avoided.